src/views/PlanTargets.vue

```diff
@@ -16,12 +16,19 @@
         <div class="field has-addons">
           <p class="control is-expanded">
             <b-field label="Latitude">
-              <b-input type="text" id="lat1" v-model="lat1" required :disabled="localtime!='X'"/>
+              <b-input type="text" id="lat1" v-model="lat1" required :disabled="observatorytime!=='X'"/>
             </b-field>
           </p>
           <p class="control is-expanded">
             <b-field label="Longitude">
-              <b-input type="text" id="lon1" v-model="lon1" required :disabled="localtime!='X'"/>
+              <b-input type="text" id="lon1" v-model="lon1" required :disabled="observatorytime!=='X'"/>
+            </b-field>
+          </p>
+        </div>
+        <div v-if="observatorytime =='X'" class="field">
+          <p class="control is-expanded">
+            <b-field label="Observatory UTC Offset (in min)">
```

Comment:
I would make this more explicit "...(in minutes)". Otherwise, maybe it is the minimum offset of multiple DST possibilities? Plus, we're not hurting for space so the layout isn't affected.

By the way, any reason you chose to use minutes instead of hours? I've usually seen timezone offsets expressed in hours (ie. we are -7 in California). Could use decimal hours to handle the weird timezone corner cases. But this is not a critical decision.